src/server/handlers/universal.js

```diff
@@ -52,6 +52,12 @@ export const handleMatch = (req, res, error, redirectLocation, renderProps) => {
       return;
     }
 
+    if (req.session.ssoDischarge) {
+      // Tell the client that it can pick up a discharge macaroon from the
+      // session and move it to local storage.
+      initialState.authStore = { hasDischarge: true };
```

Comment:
It seems that on the client we will be fetching ssoDischarge (via an action).
Can't we just set ssoDischarge here for client in initialState, so that it doesn't have to be fetched at all?
Maybe even, once sent to client in initialState it can be just immediately deleted from session (because client will already have it)?